CHANGELOG.md

```diff
@@ -11,14 +11,30 @@ accidentally triggering the load of a previous DB version.**
 * #3769 Allow ALTER TABLE DROP COLUMN on compressed hypertable
 
 **Bugfixes**
+* #3808 Properly handle `max_retries` option
```

Comment:
Line 11 is not aligned.
Line 42: please consider rewording this as
@phemmer for reporting bugs on multi-node